autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -139,7 +194,15 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
                     Warning: Be aware that utilizing refit_full models without separately verifying on test data means that the model is untested, and has no guarantee of being consistent with the original model.
             The time taken by this process is not enforced by `time_limits`.
             `cache_data` must be set to `True` to enable this functionality.
-        hyperparameters : dict
+        hyperparameters : str or dict, default = 'default'
+            Determines the hyperparameters used by the models.
+            If `str` is passed, will use a preset hyperparameter configuration.
+                Valid `str` options: ['default', 'light', 'very_light']
+                    'default': Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
+                    'light': Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
+                    'very_light': Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
+                    'toy': Results in extremely small models. Only use this when prototyping, as the model quality will be severely reduced.
+                Reference `autogluon/task/tabular_prediction/hyperparameter_configs.py` for information on the hyperparameters associated with each preset.
             Keys are strings that indicate which model types to train.
```

Comment:
To address user FAQ, we should probably add: 

"To use AutoGluon to only train some types of models, you just need to specify a subset of model-types as keys in this `hyperparameters` dict.  For example, to only train a neural network and random forest (with their default hyperparameter values) set: `hyperparameters={'NN':{}, 'RF':{}}`

Reply:
Already present in documentation:

If certain key is missing from hyperparameters, then `fit()` will not train any models of that type.
                For example, set `hyperparameters = { 'NN':{...} }` if say you only want to train neural networks and no other types of models.